src/sentry/models/organizationmemberinvite.py

```diff
@@ -77,6 +77,9 @@ class OrganizationMemberInvite(DefaultFieldsModel):
     __relocation_scope__ = RelocationScope.Organization
 
     organization = FlexibleForeignKey("sentry.Organization", related_name="invite_set")
+    # SCIM provisioning requires that the OrganizationMember object exist. Until the user
+    # accepts their invite, the OrganizationMember is a placeholder and will not be surfaced via API.
+    organization_member_id = FlexibleForeignKey("sentry.OrganizationMember")
```

Comment:
```suggestion
    organization_member = FlexibleForeignKey("sentry.OrganizationMember")
```
the postgres column will be called `organization_member_id` if we call this `organization_member`